devito/passes/iet/dtypes.py

```diff
@@ -2,11 +2,53 @@
 import ctypes
 
 from devito.ir import FindSymbols, Uxreplace
+from devito.ir.iet.nodes import Dereference
+from devito.tools.utils import as_list
+from devito.types.basic import Symbol
 
-__all__ = ['lower_complex']
+__all__ = ['lower_dtypes']
 
 
-def lower_complex(iet, lang, compiler):
+def lower_dtypes(iet, lang, compiler, sregistry):
+    """
+    Lower language-specific dtypes and add headers for complex arithmetic
+    """
+    # Include complex headers if needed (before we replace complex dtypes)
+    metadata = _complex_includes(iet, lang, compiler)
+
+    body_prefix = []  # Derefs to prepend to the body
+    body_mapper = {}
+    params_mapper = {}
+
+    # Lower scalar float16s to pointers and dereference them
+    if lang.get('half_types') is not None:
```

Comment:
Could this just be `if lang.get('half_types)`?

Reply:
It could be but I believe pep8 suggests this syntax